### ISSUE TITLE
High version client work for low version server ?

### ISSUE THREAD

**Issue by illidanpc** [chef/chef#3110]
Hi Chef,
Our chef server is using chef 11 on Red-Hat 6. Currently we want to deploy a cookbook on Oracle Enterprise Linux 7 which is similar with Red-Hat 7. I saw there's only version 12 client is work for Linux 7. So my questioin is can I use 12 client to connect 11 server. Or 11 clinet for EL6 can work for EL7?

Thanks.

**Comment by mmzyk** [comment]
@illidanpc Yes, that should be fine. While we don't promise that a newer version of the client will be backwards compatible with an older server, in this case the 12 client does work with the 11 servers, as there were no major API changes on the server from the client standpoint.

Every time we do a release, we post about it to the blog - here is the release post for Chef Client 12, where it notes it is backwards compatible with Chef Server 11. https://www.chef.io/blog/2014/12/05/release-chef-client-12-0-0/ 

So you can search the blog (the best way is to use google and specify site:chef.io/blog followed by your search term) and find these posts and see what has an hasn't change. The chef repo also has a public changelog where you can see everything that has changed with a release: https://github.com/chef/chef/blob/master/CHANGELOG.md

So in short, you're fine to use the newer client in this case, but we don't always promise that. I'm going to close this issue out, but feel free to open another issue if something else comes up.
